### ISSUE TITLE
Neptune 4 with X stepper 0.8a model and Y stepper 1.2a

### ISSUE THREAD

**Issue by friskyplatypus** [OpenNeptune3D/OpenNept4une#210]
After following the guide, I foolishly looked at the model number on the Y-Axis Stepper and determined it to be the 1.2a model. I also verified that in the printer.cfg they were set to 1.2. 

However, after installing OpenNept4une and trying to run a bed mesh there was a loud whining noise from the x-stepper. I looked at that motor to verify that I hadn't chosen the wrong model and found that it was the 0.8A model. 

Should I just modify the value in my printer.cfg and is it only the one location that I need to worry about? Namely

[tmc2209 stepper_x]
uart_pin: PB9
run_current: 1.2
interpolate: True
stealthchop_threshold: 999999

**Comment by eaman** [comment]
Yes, that is where you set the current. If you can buy an heat sink for X and Y motors.
